Blog/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,50 @@
+class PostsController < ApplicationController
+  before_action :find_post , only:[:show, :edit, :update, :destroy]
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(get_params)
+    if(@post.save)
+      redirect_to posts_path
+    else
+      render :new
```

Comment:
You probably want to pass through the rails validation errors that occur here: `@post.errors`.
The first time I tried to create a post, the body was too short, but there was no message in the UI telling me I had made a mistake.